tests/alerts/test_geomodel_alert.py

```diff
@@ -0,0 +1,138 @@
+from datetime import datetime, timedelta
+
+from freezegun import freeze_time
+
+from tests.alerts.alert_test_suite import AlertTestSuite
+from tests.alerts.negative_alert_test_case import NegativeAlertTestCase
+from tests.alerts.positive_alert_test_case import PositiveAlertTestCase
+
+import alerts.geomodel.locality as geomodel
+
+
+def _now() -> str:
+    now = datetime.utcnow()
+
+    return datetime.strftime(now, '%Y-%m-%dT%H:%M:%S.%f%z')
+
+
+class TestAlertGeoModel(AlertTestSuite):
+    alert_filename = 'geomodel_alert'
+
+    # The test cases described herein depend on some locality state being
+    # present before the tests run.
+    # The state we set up establishes a locality in San Francisco for user
+    # tester1 in which they were active a short period of time ago.  Another
+    # state is established in Toronto for user tester2.
+    # Given the states described above, detecting activity from tester1 in
+    # Toronto is expected to trigger an alert since it is a new locality that
+    # tester1 could not have travelled to.  On the other hand, no alert should
+    # fire for tester2 since both localities are in Toronto.
+
+    default_event = {
+        '_source': {
+            'sourceipaddress': '1.2.3.4',
+            'utctimestamp': _now,
```

Comment:
You can leave off this field, as it will get automatically added to any event as part of the test run.